web/libs/datamanager/src/stores/Tabs/tab_column.js

```diff
@@ -214,7 +223,7 @@ export const TabColumn = types
     },
 
     get filterable() {
-      return ((CellViews[self.type] ?? CellViews[toStudlyCaps(self.alias)])?.filterable) !== false;
+      return ((CellViews[self.type] ?? CellViews[normalizeCellAlias(self.alias)])?.filterable) !== false;
```

Comment:
lets break this down a little for readability
```suggestion
      const cellView = CellViews[self.type] ?? CellViews[normalizeCellAlias(self.alias)];
      
      return cellView?.filterable !== false;
```

Reply:
also this code implies that its possible for cellView to be undefined meaning we could get `undefined !== false` which would give us true - is that desired? do want to switch to something like 
```suggestion
      const cellView = CellViews[self.type] ?? CellViews[normalizeCellAlias(self.alias)];
      
      return !!cellView?.filterable;
```

Reply:
I can't say if that is desired. I'm not the one who what originally an author of that part of code, so I wouldn't like to change this logic to avoid unexpected results.